ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml

```diff
@@ -1340,6 +1340,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: azure-ipi-ultrassd-disks-p3-f28
+  cron: 32 18 2 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
```

Comment:
I guess we need to set `COMPUTE_NODE_TYPE: Standard_D4ps_v5` here.

Reply:
Thanks , ,made the change !!

Reply:
yes we need that as it supports ultrassds as well.